docs/ci.md

```diff
@@ -0,0 +1,152 @@
+---
+title: With CI
+description: "Test that your images build and your servers start"
+layout: docs
+---
+
+Once you've bought into container-based development,
+you'll want to set up CI to ensure your servers don't break.
+
+We think of this in two pieces:
+
+1) Run Tilt in CI to deploy to a single-use cluster.
+
+2) Run a single-use Kubernetes in CI that you can immediately throw away.
+
+Let's dig into examples of each one.
+
+## The `tilt ci` command
+
+The [`tilt ci` command](/cli/tilt_ci.html) is made for CI jobs.
+
+### How it Works
+
+Under the hood, `tilt ci`:
+
+1) Executes your Tiltfile.
+
+2) Runs all `local_resource` commands.
+
+3) Builds all images.
+
+4) Deploys all Kubernetes resources.
+
+5) Waits until all servers and other Kubernetes resources are healthy.
+
+`tilt ci` defaults to log-streaming mode. The web UI is still accessible,
+but the terminal will be a simple log stream.
+
+If any step fails, Tilt will exit immediately with an error code.
+
+If any resource crashes or looks like it will never succeed (e.g., a pod that
+takes too long to schedule), Tilt will also exit with an error code.
+
+Once all services are healthy, it will exit with status code 0.
+
+### Examples
+
+Most of our example projects use CircleCI to run `tilt ci`:
+
+<ul>
+  {% for page in site.data.examples %}
+     <li><a href="/{{page.href | escape}}#ci">{{page.title | escape}}</a></li>
+  {% endfor %}
+</ul>
+
+These each use [`ctlptl`](https://ctlptl.dev/) to set up a single-use
```

Comment:
consider a brief description of ctlptl (we do that lower down but not here?)

Reply:
hmmm...what context did you have in mind? it seems weird to say "these use ctlptl (a tool for setting up clusters) to set up a single-use cluster" (i.e., the description of what it does seems to already be here)